sigal/gallery.py

```diff
@@ -768,6 +773,9 @@ def log_func(x):
                     "defined in the settings file, which can't be serialized.",
                     exc_info=True)
                 sys.exit('Abort')
+            finally:
+                self.pool.close()
+                self.pool.join()
```

Comment:
Why moving this here ?

Reply:
If an exception gets raised (in debug mode), the pool still needs to be closed.  Otherwise you can sometimes get errors like 
```
Exception ignored in: <function Pool.__del__ at 0x7f3744d57670>
Traceback (most recent call last):
  File "/usr/lib/python3.9/multiprocessing/pool.py", line 268, in __del__
    self._change_notifier.put(None)
  File "/usr/lib/python3.9/multiprocessing/queues.py", line 378, in put
    self._writer.send_bytes(obj)
  File "/usr/lib/python3.9/multiprocessing/connection.py", line 205, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/usr/lib/python3.9/multiprocessing/connection.py", line 416, in _send_bytes
    self._send(header + buf)
  File "/usr/lib/python3.9/multiprocessing/connection.py", line 373, in _send
    n = write(self._handle, buf)
OSError: [Errno 9] Bad file descriptor
```

Reply:
Right indeed, I was focused on moving this after `terminate` but it seems fine so :+1: